src/component/ButtonDescription/ButtonDescription.js

```diff
@@ -0,0 +1,46 @@
+import {AiFillCaretUp} from 'react-icons/ai';
+import {AiFillCaretDown} from 'react-icons/ai';
+import styled from 'styled-components';
+
+import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
+
+export default function ButtonDescription({toggleDescription, showDescription, index, name, description}) {
+  //{showDescription ? 'Hide Description' : 'Show Description'}
+
+  return (
+    <Button role="button" onClick={toggleDescription}>
+      <span>
+        {showDescription === index ? (
```

Comment:
It is better to work with the id at this point instead of the index. The index order could be changed, the id is unequivocally.